### PR TITLE
round to nearest whole number for test case duration

### PR DESCRIPTION
**Pull request by madhurig**

### CODE REVIEW

**Comment by Jensen2k** [comment]
Thanks for the promptly fix, @madhurig. For referencing, this fixes #22

**Comment by jasholl** [comment]
Looks good.

File: src/agent/testresultreader.ts [hunk 1]
```diff
@@ -145,6 +145,7 @@ export class ResultReader {
 
                 //testcase duration
                 var testCaseDuration = 0; //in seconds
+                var testCaseDurationInMs = 0;
```
Comment (olivierdagenais) [inline src/agent/testresultreader.ts]: Did you mean to assign the rounded result to his variable?

Comment (madhurig) [inline src/agent/testresultreader.ts]: Oversight on my part, thought of doing it but ended up not using the variable. Will delete it.

**Pull request merged**